docs/react-native/features/tracking-error-component.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Tracking Render Errors
+description: Ensure meaningful screen names in render errors by configuring Metro bundler.
+sidebar_position: 7
+---
+
+# Tracking Render Errors
+
+To accurately track React Native rendering errors and include meaningful screen names in the `componentStack`, you need to configure your `metro.config.js` file. By default, in release builds, screen names may appear as `Unknown`. The following configuration ensures that class and function names are preserved during minification.
```

Comment:
I'd maybe rephrase to make it a bit more clear it's optional and that we do still log something by default, maybe:


> If the Embrace SDK detects a component stack trace in a React Native rendering error it will log it automatically. By default, in release builds, screen names may appear as `Unknown` as in the following example:
-->example from below

> It is possible to configure these names to be preserved during minification to make it easier to debug rendering errors in release builds at the cost of a slightly larger bundle size:
-->example from below